src/components/shared/Header.tsx

```diff
@@ -30,156 +33,191 @@ export const Header = () => {
         {Translate('Data')}
       </Link>
-    </Dropdown.Item>
-    <Dropdown.Item>
+    </div>
+    <div className={getTabClass('/Publications')}>
       <Link to={withLocaleUrl("Publications")}>
         {Translate('Publications')}
       </Link>
-    </Dropdown.Item>
-    <Dropdown.Item>
+    </div>
+    <div className={getTabClass('/About')}>
       <Link to={withLocaleUrl("About")}>
         {Translate('About')}
       </Link>
-    </Dropdown.Item>
-    <Dropdown.Item>
-      <Accordion fluid>
-          <Accordion.Title
-            active={partnershipDrownDownActive}
-            index={0}
-            onClick={handleClick}
-            className={'flex center-item py-0'}
-          >
-            {Translate("Partnerships")}
-            <Icon name='dropdown' className={`ml-5 ${partnershipDrownDownActive ? 'dropdown__icon' : 'dropdown__icon--active'} `}/>
-          </Accordion.Title>
-          {renderPartnershipsAccordianContent(partnershipDrownDownActive)}
-        </Accordion>
-    </Dropdown.Item>
-    <Dropdown.Item>
-      <div onClick={() => changeLanguages()}>
-          {language === LanguageType.english ? "FR" : "EN" }
-      </div>
-    </Dropdown.Item>
-  </Dropdown.Menu>
-</Dropdown>)
-}
+    </div>
+    <div className={'nav__item col-auto h-100 flex center-item'}>
+      <Dropdown text={Translate("Partnerships")}>
+        {renderPartnershipsDropDownMenu()}
+      </Dropdown>
+    </div>
+    <div className={'col-auto h-100 px-0'}>
+      |
+    </div>
+    <div className={"nav__item col-auto h-100 flex center-item cursor"} >
+    <Dropdown text={language.toUpperCase()}>
+        {renderLanguageDropDownMenu()}
+      </Dropdown>
+    </div>
+  </div>)
+  }
+  
+  const renderPartnershipsDropDownMenu = () => (
+    <Dropdown.Menu>{
+      PartnershipsConfig.map( p => (
+        <Dropdown.Item>
+          <Link to={withLocaleUrl(`Partnerships/${p.routeName}`)}>{Translate("PartnershipsList", [p.routeName])}</Link>
+        </Dropdown.Item>
+      ))}
+        <Dropdown.Item>
+          <Link to={withLocaleUrl('Unity')}>{Translate("WhoUnity")}</Link>
+        </Dropdown.Item>
+    </Dropdown.Menu>)
+  
+  const renderLanguageDropDownMenu = () => (
+    <Dropdown.Menu >
+      {state.language !== LanguageType.english ?   
+        <Dropdown.Item onClick={() => {changeLanguages(LanguageType.english)}}>
+          { LanguageType.english.toUpperCase() }
+        </Dropdown.Item> : ""}
+     {state.language !== LanguageType.french ?        
+        <Dropdown.Item onClick={() => {changeLanguages(LanguageType.french)}}>
+        { LanguageType.french.toUpperCase() }
+        </Dropdown.Item> : ""}
+     {state.language !== LanguageType.german ?      
+         <Dropdown.Item onClick={() => {changeLanguages(LanguageType.german)}}  >
+        { LanguageType.german.toUpperCase() }
+        </Dropdown.Item> : ""}
+    </Dropdown.Menu>)
+  
+  const renderPartnershipsAccordianContent = (partnershipDrownDownActive: boolean) => (
```

Comment:
What is the difference between this method and the renderParnershipDropdownMenu method?

Reply:
Is one for desktopnav and the other for mobilenav? Could there be a better way to do that instead of 2 very similar methods (accordian vs dropdown is the only difference right)?

Reply:
i didn't set up these methods, but I think it makes sense to have them separate since the content is pretty different between the 2